src/docs/src/ddocs/nouveau.rst

```diff
@@ -606,8 +609,8 @@ using a single ``if`` statement.
 .. code-block:: javascript
 
     if (typeof doc.town == "string" && typeof doc.name == "string") {
-        index("string", "town", doc.town);
-        index("string", "name", doc.name);
+        index("string", "town", doc.town, { facet: true });
```

Comment:
facet:true isn't recognised, all strings and doubles are indexed for faceting now.